docs/data-workflow.md

```diff
@@ -0,0 +1,130 @@
+*   KenW vets these files and posts them to https://www.unicode.org/Public/draft/UCD/ucd/ .
+*   A unicodetools GitHub contributor fetches these files, preprocesses the contents of Unihan.zip,
+    and creates a pull request as for “regular” data files.
+    (The processed data files go into .../unicodetools/data/ucd/dev/Unihan.)
+
+### NamesList.txt
+
+*   Iterate between KenW and Michel.
+*   Generated from UnicodeData.txt and an annotations file, using some C program.
+*   Used for generating code charts.
+*   KenW posts NamesList.txt into https://www.unicode.org/Public/draft/UCD/ucd/ .
+*   A unicodetools GitHub contributor fetches this file
+    and creates a pull request as for “regular” data files.
+
+### Folder readmes
+
+The “source of truth” for these is outside of GitHub for now.
+KenW updates or vets these files and posts them to https://www.unicode.org/Public/draft/ .
+A unicodetools GitHub contributor fetches these files and creates a pull request as above.
+
+See https://github.com/unicode-org/properties/issues/8 “simplify versioning of readme files”
+
+### “Regular” data files
+
+Changes are made in a GitHub pull request.
+*   Contributors who don’t use GitHub or don’t contribute to the unicodetools project
+    can send changes (updated files or deltas) to someone who is comfortable using GitHub.
+*   Updated files could be shared in various ways including via email or via private FTP areas.
+*   Updated files should be based on the latest (or fairly recent) data in the unicodetools repo.
+*   Updated files should not be posted directly to https://www.unicode.org/Public/...
+
+Pull request cycle:
+*   One commit for manual or contributed data changes.
+*   Another commit for tool code updates, generated/derived data files,
+    and for fixing test failures (continuous-integration tests or manual tests).
+*   Peer review of the changes.
+*   QA may include pushing files to a private area, e.g., for KenW to review.
+*   More commits as necessary until changes look good, tests pass,
+    and someone (preferably KenW) approves.
+*   **Squash**-and-merge.
+
+### Notes about emoji data files
+
+One difference here: Multiple stages, generating charts along the way.
+Initial draft versions of eg annotations for candidates,
+which is in addition to the current data for the next version of Unicode.
+Once they get code points, they go into the regular files.
+The emoji tools also read annotation data from CLDR and from the candidates file,
+and use the CLDR emoji collation data (and interpolates the candidates data).
+
+Another difference:
+The charts are not normative; they get updated out of cycle, for example with new vendor images.
+
+https://github.com/unicode-org/unicodetools/tree/main/unicodetools/data/emoji/dev
+
+## Publication
+
+Certain snapshots of the .../dev/ files are copied into https://www.unicode.org/Public/draft/
+for Unicode alpha, beta, and final releases, and more as appropriate.
```

Comment:
This seems to imply less frequent updates to files to alpha/beta directories, which may mean repeated bug reports for an issue that's already fixed in the development branch. I would prefer a more "live" alpha/beta, where the contents of the alpha/beta directory are kept very close to the latest development branch.

Reply:
The main goal is to keep data files more consistent with each other. In the past, if you didn't pick up a beta snapshot early, you had a good chance of files being updated piecemeal, for example with core and derived files out of sync. That might have been ok for humans looking at files, but not useful for people and projects downloading the set and trying to work with it.

This should definitely result in less frequent changes in the public folders, but it does not preclude us publishing whole consistent snapshots several times as we feel useful.